doc/en/_static/pytest.css

```diff
@@ -0,0 +1,11 @@
+/* Hide list bullets and top-level indent in TOC sidebar */
+.sidebar-scroll ul {list-style: none;}
+.sidebar-scroll > ul {padding-left: 0;}
+.sidebar-scroll li {margin: 0.4em 0;}
```

Comment:
```suggestion
.sidebar ul {
   list-style: none;
   padding-left: 0;
}
.sidebar ul li {
    margin: 0.4em 0;
}
```

Reply:
I'm gonna say this isn't necessary because I clearly misunderstood what the goals were here.